docs/api/spec.md

```diff
@@ -378,7 +381,7 @@ For example, to raise a specific intent:
 
 ```ts
 try {
-  const resolution = await fdc3.raiseIntent('StageOrder', context);
+  const resolution = await fdc3.raiseIntent("StageOrder", context);
```

Comment:
If you search for:
 ){
in the example code below this line, you'll find (I think) 3 matches.

i.e. missing a space. These code block should show:
 ) {